### ISSUE TITLE
Rosegarden not starting

### ISSUE THREAD

**Issue by mawis** [NixOS/nixpkgs#76639]
**Describe the bug**
The `rosegarden` package seems to be broken and cannot be started.

```
matthias:~/ $ export QT_DEBUG_PLUGINS=1               
matthias:~/ $ rosegarden
QFactoryLoader::QFactoryLoader() checking directory path "/home/matthias/lib/qt-5.12.3/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/run/wrappers/lib/qt-5.12.3/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/home/matthias/.nix-profile/lib/qt-5.12.3/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/etc/profiles/per-user/matthias/lib/qt-5.12.3/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/nix/var/nix/profiles/default/lib/qt-5.12.3/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/run/current-system/sw/lib/qt-5.12.3/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/nix/store/b795rcbv3wz8m8ds46pifayj0lm616ll-fcitx-with-plugins-4.2.9.6/lib/qt4/plugins/platforms" ...
QFactoryLoader::QFactoryLoader() checking directory path "/nix/store/8z5s6vkr4kcm3j5972mzrhy7dxhgh1jc-rosegarden-19.06/bin/platforms" ...
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

[1]    5785 abort (core dumped)  rosegarden
```

**To Reproduce**
Happens always when `rosegarden` gets started. I start it from the command line to see the console output.

**Expected behavior**
Rosegarden should start and open a window.

**Metadata**
- system: `"x86_64-linux"`
 - host os: `Linux 4.19.90, NixOS, 19.09.1629.ce54d9601ea (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - channels(root): `"nixos-19.09.1629.ce54d9601ea, unstable-20.03pre204199.3140fa89c51"`
 - channels(matthias): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
attribute:
- rosegarden
module:
```

**Comment by andersk** [comment]
Fix sent as #76799.

**Comment by mawis** [comment]
@andersk thanks dir the fix
